base/OpenGLVisitor.cpp

```diff
@@ -132,6 +120,34 @@ void OpenGLVisitor::visit(Group* g) {
 #endif
             vector<int>::iterator indexIter = (*polyIter)->m_vIndex.begin();
             vector<int>::iterator indexEnd = (*polyIter)->m_vIndex.end();
+            #ifdef HAVE_OPENGLES
```

Comment:
I had in mind to duplicate OpenGLVisitor in OpenGLESVisitor it could be easier to change renderer at runtime eventually ?

Reply:
Yes, why not, the code will certainly be better (I'm not a C++ expert!). But to change renderers at runtime, I think OpenGL libraries need to be loaded dynamically.